Nokia/drg/res/values/config.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<bool name="config_swipe_up_gesture_setting_available">true</bool>
+    <bool name="config_supportSystemNavigationKeys">true</bool>
+    <bool name="config_smart_battery_available">true</bool>
+    <bool name="config_supportDoubleTapWake">true</bool>
+    <bool name="config_showNavigationBar">true</bool>
+    <bool name="config_enableMultiUserUI">true</bool>
+    <bool name="config_camera_sound_forced">false</bool>
+    <bool name="config_dozePulsePickup">true</bool>
+    <bool name="config_display_rat">false</bool>
+    <bool name="config_useRoundIcon">true</bool>

```

Comment:
That's a ROM decision, not hardware decision.
But anyway, this value is obsolete and used by noone.